extension/ackextension/config.go

```diff
@@ -0,0 +1,10 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ackextension // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/ackextension"
+
+// Config defines configuration for ack extension
+type Config struct {
+	// StorageType defines the storage type of the extension. Currently planned for In memory type. Future consideration is disk type.
+	StorageType string `mapstructure:"StorageType,omitempty"`
```

Comment:
Can we use `storage` option instead pointing to a storage extension similar to what we do in https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/exporterhelper#persistent-queue. If not provided, memory storage is used.

Reply:
Okay. I changed the type to `*component.ID`